libres/lib/enkf/ecl_config.cpp

```diff
@@ -367,12 +367,16 @@ static void handle_has_end_date_key(ecl_config_type *ecl_config,
                                     const config_content_type *config) {
     const char *date_string = config_content_get_value(config, END_DATE_KEY);
     time_t end_date;
-    if (util_sscanf_date_utc(date_string, &end_date))
+    bool end_date_parsed_ok;
+    end_date_parsed_ok = util_sscanf_isodate(date_string, &end_date);
```

Comment:
Any reason to split declaration and definition over two lines?

Reply:
Nope.. fixed.